specification/security/resource-manager/Microsoft.Security/stable/2023-01-01/pricings.json

```diff
@@ -314,29 +315,11 @@
     },
     "OperationStatus": {
       "type": "object",
-      "description": "A status describing the success/failure of the extension's enablement/disablement operation.",
+      "description": "A status describing the success/failure of the enablement/disablement operation.",
       "properties": {
         "code": {
           "type": "string",
-          "description": "The operation status code.",
-          "enum": [
-            "Succeeded",
-            "Failed"
-          ],
-          "x-ms-enum": {
```

Comment:
This was removed since `OperationStatus` is a global definition which causes conflicts in SDK generation - changed to a relaxed form which is compatible with both